README.md

```diff
@@ -1,3 +1,19 @@
 # CoreDNS
 
-CoreDNS is a DNS server.
+CoreDNS is a DNS  server. And this is a Kubernetes Charm to deploy CoreDNS on Kubernetes with Juju.
+
+## How to build the charm
+
+Make sure you've got the [coredns-interface](https://github.com/DomFleischmann/coredns-interface) in your $CHARM_INTERFACES_DIR and simply build the charm with:
+``` charm build charm-coredns -o ..```
+
+## How to deploy the charm
+
+For deploying the charm you will have to have a kubernetes cluster bootstrapped with juju to be able to deploy kubernetes charms. Than you will simply need to execute the following in the charm builds dir:  
```

Comment:
"you'll need a Kubernetes cluster bootstrapped with Juju" - let's find and add a link for people who don't know how to do this.